app/views/publisher_information/how_to_use_publisher.erb

```diff
@@ -0,0 +1,14 @@
+<% content_for :browser_title, t("publisher_information.how_to_use_publisher.title") %>
```

Comment:
I think the file extension is conventionally '.html.erb'.

Reply:
Ah, thanks for that I've changed it.